libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SmartSyncLogger.java

```diff
@@ -213,6 +213,16 @@ public static void v(String tag, String msg, Object obj) {
         v(tag, msg + ": " + objStr);
     }
 
+    /**
+     * Sets the log level to be used.
+     *
+     * @param level Log level.
+     */
+    public static void setLogLevel(SalesforceLogger.Level level) {
```

Comment:
True. I like your approach of adding a method to set log level better. It's more obvious and easier to document. I'll send a PR.